### ISSUE TITLE
ncu exits with error code 1 with no upgrades

### ISSUE THREAD

**Issue by meganwalker-ibm** [raineorshine/npm-check-updates#1137]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 14`

---

## Steps to Reproduce
```
$ node -v
v14.18.3
$ npm -v
8.12.1
$ npx ncu --version
13.1.4
$ npx ncu --errorLevel 2 --minimal
Using config file /Users/megan.walker/firefly/control-plane/.ncurc.js
Checking /Users/megan.walker/firefly/control-plane/package.json
[====================] 11/11 100%

All dependencies match the desired package versions :)

Run npx npm-check-updates -u to upgrade package.json

Dependencies not up-to-date
$ echo $?
1
```

## Current Behavior
Returns an error code, but does not say which dependencies aren't up to date

## Expected Behavior
The same table as previously is printed out i.e.
```
 express           4.12.x  →   4.13.x
 multer            ^0.1.8  →   ^1.0.1
 react-bootstrap  ^0.22.6  →  ^0.24.0
 react-a11y        ^0.1.1  →   ^0.2.6
 webpack          ~1.9.10  →  ~1.10.5
```
I'm not sure why it's saying that `All dependencies match the desired package versions :)` when my package.json is specifying an old version of standard.

**Comment by raineorshine** [comment]
Wait a second... I just noticed you're specifying `--minimal`. It looks like it is behaving correctly. All of your upgrades are within the specified semver ranges. Is that not your intention by specifying `--minimal`?

**Comment by raineorshine** [comment]
I'm pretty confident it was just the error code that was incorrect. I made a fix in 4aaf2fe and published to `v13.1.5`. Please give it a try!

**Comment by meganwalker-ibm** [comment]
Hm you're right - all our packages are iwthin semver, aside from one which is ignored in the .ncurc...

But if that's the case why are we receiving an error return code, rather than a 0 return code?
